feature-libs/asm/customer-360/components/sections/asm-customer-promotion/asm-customer-promotion.model.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Customer360Promotion } from '@spartacus/asm/customer-360/root';
+
+export interface PromotionEntry extends Customer360Promotion {
+  applied: boolean;
```

Comment:
seems `PromotionEntry`and `Customer360Promotion ` are having exactly seem params. Can we just use one of them?